xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -104,7 +106,7 @@ enum AppAction: CustomLogStringConvertible {
     /// Set sphere/user nickname
     /// Sets form field, and persists if needed.
     case setNickname(_ nickname: String)
-    case persistNickname(_ nickname: String)
+    case submitNickname(_ nickname: Petname.Name)
```

Comment:
Updated to more clearly reflect how it's being used, mirrors the way GatewayURL is managed